app/controllers/admin/contributions_controller.rb

```diff
@@ -30,7 +29,11 @@ def set_title
     @title = t("admin.contributions.index.title")
   end
 
+  def resource
+    Payment.find params[:id]
```

Comment:
Yeah, this is a bit weird. But we are not using the contribution here anymore. The collection method now uses the ContributionDetail which is basically ```contributions JOIN payments```. I used the Payment as a resource so I would not have to proxy every Payment method through ContributionDetail.